source/GameInterface.Tests/Serialization/SerializerTests/EquipmentSerializationTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using GameInterface.Serialization.Impl;
+using GameInterface.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.Core;
+using Xunit;
+using System.Reflection;
+using static TaleWorlds.Core.Equipment;
+
+namespace GameInterface.Tests.Serialization.SerializerTests
+{
+    public class EquipmentSerializationTest
+    {
+        [Fact]
+        public void Equipment_Serialize()
+        {
+            Equipment equipment = new Equipment();
+
+            BinaryPackageFactory factory = new BinaryPackageFactory();
+            EquipmentBinaryPackage package = new EquipmentBinaryPackage(equipment, factory);
+
+            package.Pack();
+
+            byte[] bytes = BinaryFormatterSerializer.Serialize(package);
+
+            Assert.NotEmpty(bytes);
+        }
+
+        [Fact]
+        public void Equipment_Full_Serialization()
+        {
+            Equipment equipment = new Equipment();
+            FieldInfo _equipmentType = typeof(Equipment).GetField("_equipmentType", BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public);
```

Comment:
Update this to be similar to https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/taverns-serialization/source/GameInterface.Tests/Serialization/SerializerTests/CharacterSkillsSerializationTest.cs#L29